README.md

```diff
@@ -67,6 +68,14 @@ To-do List
 
 [open an issue]: https://github.com/J3RN/tree-sitter-gleam/issues/new
 
+Contributing
+------------
+
+1. Change files such as `grammar.json` and `queries/highlight.scm`.
+2. The grammar needs to be generated from the `grammar.js` file by running `npm run generate`.
+3. Add lang feature tests in `test/corpus/functions.txt`.
+4. Run `npm run test` and fix tests in `test/corpus/functions.txt`.
```

Comment:
`functions.txt` isn't the _only_ place for tests; there's a bunch of files in `test/corpus/` that exist for testing different bits of the language.  How about this?

```suggestion
3. Add parser feature tests to the relevant file(s) in `test/corpus/` (or make a new one!).
4. Run `npm run test` and fix any failing tests.
```